src/index/CompressedRelation.h

```diff
@@ -783,7 +783,7 @@ class CompressedRelationReader {
   // index scans between joining them to get better estimates for the beginning
   // and end of incomplete blocks.
   std::optional<ScanSpecAndBlocksAndBounds::FirstAndLastTriple>
-  getFirstAndLastTriple(
+  getFirstAndLastTripleIgnoringGraph(
```

Comment:
Also extend the comment s.t. it states that the graph filter of the `metadataAndBlocks` is ignored if specified, so we always get the first and last triple in ANY graph.